scripts/biospecimen_count.Rmd

```diff
@@ -161,33 +167,68 @@ if ( any(is.na(rna_counts$stage)) ) {
 
 <br>
 
-#### Combine Count Tables
+### Combine Count Tables
 
 Combine the count tables, organize them for display, and save.
 
----
+```{r}
+rbind(dna_counts, rna_counts) %>%
```

Comment:
bind_rows() is now preferred as rbind() is deprecated

Reply:
It doesn't matter to me which one I use, but `rbind()` is a base function, where have you seen that it's deprecated?

Reply:
oh, my mistake it is rbind_all from dplyr, so OK to use this here!

>rbind_list() and rbind_all() have been deprecated. Instead use bind_rows().